x/distribution/client/common/common.go

```diff
@@ -0,0 +1,143 @@
+package common
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	distr "github.com/cosmos/cosmos-sdk/x/distribution"
+)
+
+// QueryParams actually queries distribution params.
+func QueryParams(cliCtx context.CLIContext, queryRoute string) (PrettyParams, error) {
+	route := fmt.Sprintf("custom/%s/params/community_tax", queryRoute)
+
+	retCommunityTax, err := cliCtx.QueryWithData(route, []byte{})
+	if err != nil {
+		return PrettyParams{}, err
+	}
+
+	route = fmt.Sprintf("custom/%s/params/base_proposer_reward", queryRoute)
+	retBaseProposerReward, err := cliCtx.QueryWithData(route, []byte{})
+	if err != nil {
+		return PrettyParams{}, err
+	}
+
+	route = fmt.Sprintf("custom/%s/params/bonus_proposer_reward", queryRoute)
+	retBonusProposerReward, err := cliCtx.QueryWithData(route, []byte{})
+	if err != nil {
+		return PrettyParams{}, err
+	}
+
+	route = fmt.Sprintf("custom/%s/params/withdraw_addr_enabled", queryRoute)
+	retWithdrawAddrEnabled, err := cliCtx.QueryWithData(route, []byte{})
+	if err != nil {
+		return PrettyParams{}, err
+	}
+
+	return NewPrettyParams(retCommunityTax, retBaseProposerReward,
+		retBonusProposerReward, retWithdrawAddrEnabled), nil
+}
+
+// QueryParams queries delegator rewards. If valAddr is empty string,
+// it returns all delegations rewards for the given delegator; else
+// it returns the rewards for the specific delegation.
+//
+// TODO refactor remove this function, we should likely not clump these two
```

Comment:
Why not just do this now?

Reply:
Done. localnet failure was just a glitch